tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,16 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
+    assert nobodys_item not in fatimah.inventory
+    assert len(fatimah.inventory) == 3
+    assert len(jolie.inventory) == 0
```

Comment:
`assert jolie.inventory == []` would ensure that the `inventory` is an empty list.